README.md

```diff
@@ -192,6 +192,26 @@ This is the only harness that uses `run_benchmark`'s argument, `num_itrs_hint`.
 ./run_benchmarks.rb --rss
 ```
 
+## Rendering a graph
+
+`--graph` option of `run_benchmarks.rb` allows you to render benchmark results as a graph.
+
+```
+./run_benchmarks.rb --graph
+```
+
+### Installation
+
+Before using this option, you might need to install the dependencies of [Gruff](https://github.com/topfunky/gruff):
+
+```bash
+# macOS
+brew install imagemagick
+
+# Ubuntu
+sudo apt-get install libmagickwand-dev
+```
```

Comment:
I don't like adding extra dependencies, and from experience, imagemagick can be a pain to install on some platforms. Will allow it because it's useful.